src/attachments/downloader.js

```diff
@@ -124,7 +125,11 @@ function copyFile(from, to) {
 export function download(url, path, headers) {
   return new Promise((resolve, reject) => {
     const parsedUrl = nodeUrl.parse(url);
-    http
+    let client = http;
```

Comment:
`const client = parsedUrl.protocol === "https:" ? https : http;`
To get rid of `let`